examples/conf-quickstart/druid/_common/common.runtime.properties

```diff
@@ -116,3 +116,8 @@ druid.selectors.coordinator.serviceName=druid/coordinator
 druid.monitoring.monitors=["com.metamx.metrics.JvmMonitor"]
 druid.emitter=logging
 druid.emitter.logging.logLevel=info
+
+# Storage type of double columns
+# ommiting this will lead to index double as float at the storage layer
+
+druid.indexing.doubleStorage=double
```

Comment:
Newline at end of file please.

Reply:
And this property should be documented. How about on `docs/configuration/index.md` in a new section near the bottom?